db/models.py

```diff
@@ -14,3 +14,45 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True)
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
better name it `movies`:
```suggestion
    genres = models.ManyToManyField(Genre, related_name="movies")
```